src/utils/mtev_stacktrace.c

```diff
@@ -426,6 +426,11 @@ mtev_dwarf_walk_map(void (*f)(const char *, uintptr_t)) {
 #else
 #endif
 }
+#else
```

Comment:
How are these changes related to OpenSSL ..?

Reply:
They aren't, I'll separate them. (same PR, but separate commit)